### ISSUE TITLE
CSS: seeing hostages causes graphics bug

### ISSUE THREAD

**Issue by knacki-gizmo** [ValveSoftware/Source-1-Games#1930]
Game: 
Counterstrike Source

System:
Mac OS El Capitan 10.11.1
MacBook Pro 13", middle 2012
Intel HD Graphics 4000 1536 MB
2,5 GHz Intel Core i5
seriesnr: C1MP4P02DTY3

When i enter a map with hostages, cs_office or cs_italy for example,
i move to the hostages so i can see them, a hard graphics bug appears and it blocks my view completely. 
If I dont look at them, there is no graphics bug.

Yours sincerely,
F. B.

**Comment by knacki-gizmo** [comment]
![bildschirmfoto 2015-11-01 um 20 50 47](https://cloud.githubusercontent.com/assets/15196277/10871008/8eeb5606-80da-11e5-8b2b-09aecb522d41.png)
![Uploading Bildschirmfoto 2015-11-01 um 20.50.48.png…]()
![Uploading Bildschirmfoto 2015-11-01 um 20.50.55.png…]()

**Comment by keehun** [comment]
YES, this has been a problem for the last month or two. Insanely annoying.

**Comment by keehun** [comment]
More discussion here: https://steamcommunity.com/app/240/discussions/0/490121928350816862/

It seems everyone in the thread is experiencing the same issue.

**Comment by beckyconning** [comment]
I have the same issue.

**Comment by rogual** [comment]
Same issue here. This renders all cs_ maps unplayable.

OS X 10.11.2
MacBook Pro (Retina, 13-inch, Mid 2014)
Intel Iris 1536 MB

![screenshot](http://i.imgur.com/9wQnJmp.jpg)

**Comment by hydex80** [comment]
Also same issue here. Hoped that it was solved with the new counterstrike source update.. but it isn't!
My system
OSX 10.11.2
Imac 21,5 inch medium 2011
AMD Radeon HD 6750M 512 MB.

It really makes the hostage maps unplayable. :( please solve this.

**Comment by pascalbaljet** [comment]
Same here. Macbook Air 13" (mid 2013), OS X 10.11.4.

**Comment by philip-lamb** [comment]
Same here

Hardware Overview:

  Model Name:	Mac Pro
  Model Identifier:	MacPro6,1
  Processor Name:	Quad-Core Intel Xeon E5
  Processor Speed:	3.7 GHz
  Number of Processors:	1
  Total Number of Cores:	4
  L2 Cache (per Core):	256 KB
  L3 Cache:	10 MB
  Memory:	16 GB
  Boot ROM Version:	MP61.0116.B17
  SMC Version (system):	2.20f18
  Illumination Version:	1.4a6

AMD FirePro D300:

  Chipset Model:	AMD FirePro D300
  Type:	GPU
  Bus:	PCIe
  Slot:	Slot-2
  PCIe Lane Width:	x16
  VRAM (Total):	2048 MB
  Vendor:	ATI (0x1002)
  Device ID:	0x6810
  Revision ID:	0x0000
  ROM Revision:	113-C4012F-687
  gMux Version:	4.0.11 [3.2.8]
  EFI Driver Version:	01.00.687

**Comment by morozovm** [comment]
+1

**Comment by goganchic** [comment]
I'm afraid no one who can fix this bug reading this ticket :(

**Comment by philip-lamb** [comment]
The problem is that the original reporter did not use the correct syntax for the subject line (needed to be [CS:S], not CSS), and did not assign the label:"Counter-Strike: Source".

**Comment by raymusmunt** [comment]
This is also affecting me.  OSX 10.11.6 Mac Pro 2008 - ATI Radeon HD 5870

**Comment by enkaskal** [comment]
I've opened a steam support incident in the hopes it will help. 

Ticket #: 4918-ERTA-1881

**Comment by georgealton** [comment]
also seeing this issue

**Comment by rememberedbound** [comment]
Same bug in Half Life 2 deathmatch. It's the NPC models, specifically the faces, the rest of them renders fine. Whatever you're using to morph the face animations is catastrophically failing. I left a demo screen shot on the other bug report.

**Comment by byrondevonwall** [comment]
also seeing this issue

**Comment by housebolt** [comment]
+1

**Comment by dmi-try** [comment]
Please fix it. Lots of macOS users cannot play maps with hostages since 16th Sep 2015. Here is giant thread about it on community forum: https://steamcommunity.com/app/240/discussions/0/490121928350816862/

**Comment by rmmoul** [comment]
This is still an issue, and has been on both my mac air, and my imac. Why hasn't this very reproducible bug been taken care of?

**Comment by rogual** [comment]
I guess they want us to buy their newer games instead of playing their old ones. Clever way of going about it, breaking the old game with an update.

Poor show by Valve.

**Comment by hydex80** [comment]
I reported this issue one year ago still not Fixed. Shame on you valve i bought this game payed for it can i have my money back? Or are you guys fix it please give a answer if you reject Or accept this issue then we know what to do

**Comment by Kafkalasch** [comment]
As a workaround you can try this skin: https://files.gamebanana.com/skins/fem_hostage.zip (just unpack its contents into cstrike/custom ). Since i am using this skin i haven't had any problems at all. (credits go to Explorer94 who - to my best knowledge - originally posted this solution here https://steamcommunity.com/app/240/discussions/0/490121928350816862/?ctp=14 )

**Comment by Zola** [comment]
@Kafkalasch many thanks , it works! 
downloading https://files.gamebanana.com/skins/fem_hostage.zip 
put fem hostage.vpk to : 
~/Library/Application Support/Steam/steamapps/common/Counter-Strike Source/cstrike/custom 
finally works.

**Comment by uyouthe** [comment]
@Kafkalasch @Zola just put that file into game and now I have icons broken, it displays A for health and B for armor, letters instead of icons

**Comment by jonhyleo** [comment]
@Zola Thanks a lot. Problem solved.

**Comment by hansblix** [comment]
@Zola Thanks! It works!!!!

**Comment by NOTANO0B** [comment]
2020 and this is still an issue. any fixes?

**Comment by rmmoul** [comment]
Use a different hostage skin. If you google around some there’s a skin with female hostages that works since the devs have straight up just not given a shit about the people who play the game.


--
Raymond Moul

707-276-6685
rmmoul@gmail.com

**Comment by ljcool2006** [comment]
permanent(?) link to fem hostage skin: https://gamebanana.com/mods/214226

**Comment by TotallyARussian** [comment]
i wonder if the console command r_flex 0 could be used to fix this, anyone on mac tried that? try it with r_eyes 0 too if just the flex command won't work, the first command turns off the flexes which seem to be causing the problem but i'm not sure if the eyes do, if they do then r_eyes 0 can be used to turn them off as well, if the commands work you can use them as a temporary fix instead of having to download custom hostage models that don't have flexes
